docs/references.bib

```diff
@@ -281,6 +281,13 @@ @InProceedings{   fuerer-lochbihler-schneider-traytel2020
   bibsource     = {dblp computer science bibliography, https://dblp.org}
 }
 
+@Inproceedings{   Gallier2011Notes,
```

Comment:
This was what "semantic scholar" suggested, but I have no idea what the right choice is.

Reply:
LGTM as far as `doc-gen` is concerned.